src/views/Bond/BondRedeem.tsx

```diff
@@ -108,14 +108,14 @@ function BondRedeem({ bond }: { bond: IAllBondData }) {
             balance={<DisplayBondDiscount key={bond.name} bond={bond} />}
             isLoading={isBondLoading}
           /> */}
-          <div className="data-row">
+          <Box display="flex" flexDirection="row" justifyContent="space-between">
```

Comment:
just FYI this is temporary. We'll add support for ReactElement to to component library so this entire element can use the datarow component. Didn't realize this was an issue until i saw the inline comment